build/yaml/botbuilder-js-daily.yml

```diff
@@ -0,0 +1,259 @@
+#
+# This runs under BotBuilder-JS-daily-yaml.
+#
+
+# "name" here defines the build number format. Build number is accessed via $(Build.BuildNumber)
+name: $(Build.BuildId)
+
+pool:
+  vmImage: 'macOS-10.15'
+
+variables:
+  NodeVersion: 12.x
+#  PackageVersion: Define this in Azure to be settable at queue time. Ex: 4.11.0-dev.{DateStamp}.{CommitHash} 
+#  PreviewPackageVersion: Define this in Azure to be settable at queue time. Ex: 4.11.0-dev.{DateStamp}.{CommitHash}
+
+steps:
+- powershell: |
+   # Replace {DateStamp} and {CommitHash} tokens with the actual values in vars PackageVersion and PreviewPackageVersion
+   $dateStamp = (Get-Date -format "yyyyMMdd");
+   $commitHash = "$(Build.SourceVersion)".SubString(0,7);
+   
+   "Raw PackageVersion = $(PackageVersion)";
+   $v = "$(PackageVersion)".Replace("{DateStamp}",$dateStamp).Replace("{CommitHash}",$commitHash);
+   Write-Host "##vso[task.setvariable variable=PackageVersion;]$v";
+   "Resolved PackageVersion = $v";
+   
+   "Raw PreviewPackageVersion = $(PreviewPackageVersion)";
+   $ppv = "$(PreviewPackageVersion)".Replace("{DateStamp}",$dateStamp).Replace("{CommitHash}",$commitHash);
+   Write-Host "##vso[task.setvariable variable=PreviewPackageVersion;]$ppv";
+   "Resolved PreviewPackageVersion = $ppv";
+  displayName: 'Resolve package version variables'
+
+- template: js-build-steps.yml
+
+- task: Npm@1
+  displayName: 'npm pack adaptive-expressions'
```

Comment:
Can we use `lerna exec npm pack` to automatically pack everything referenced in the `lerna.json` file? Single source of truth would be nice 👍

Reply:
Good idea. That is outside the scope of this PR, so I have created Issue #2908 for that and assigned it to myself. https://github.com/microsoft/botbuilder-js/issues/2908